README.md

```diff
@@ -108,14 +117,49 @@ Both command line scripts and Docker Image resources can be parametrised by usin
+| -alias                | Key alias                    | Any string, by default the same as value of `servicename` parameter        |
+| -role                 | Role to be fulfilled by the keystore key, different roles correspond to dedicated settings in openssl.cnf file | `client`, `server`, `both`, by default `both`        |
+| -rootcapass           | Password set for Root CA, is required for signing the additional keystores | Any string. Lack of this parameter will result with an exception.       |
+| -keysize              | RSA key length               | `2048`, `4096`..., by default `2048`   |
+| -keystoretype         | Type of the keystores (containing private keys)  | `PKCS12`, `JKS`, `JCEKS`, by default `JCEKS` |
+| -keystorepass         | Password for the keystores   | Any string between 6 to 1023 characters, if not provided a prompt will be displayed                  |
+| -notruststore         | Flag for blocking truststore generation | N/A, providing the flag turns off truststore generation           |
+| -truststoretype       | Type of the truststores (containing public keys) | `JKS`, `JCEKS`, by default `JCEKS`           |
+| -truststorepass       | Password for the truststores | Any string between 6 to 1023 characters, if not provided a prompt will be displayed                  |
+| -certdname            | Distinguished Name of the CA certificate, starting with slash and quoted | "/C=GB/ST=UK/L=Maidenhead/O=Alfresco Software Ltd./OU=Unknown/CN=Custom Service" |
+| -servername           | DNS Name for CA Server       | Any string, `localhost` by default        |
+| -alfrescoformat       | Default format for certificates, truststores and keystores | `classic` or `current` (only supported from ACS 7.0) |
 
 When using Alfresco on an internal network, each server should have a different name. This names can be configured on the parameters named as `*servername`. In order to avoid browser complains about certificates, it's recommended to include the name of the server as `Alternative Name` in the certificate. This should be at least required for SOLR Web Console, as this application is only available in `https` when using this configuration. If you are working under a Web Proxy, use the name of this proxy for the `*servername` parameters.
+Scripts have been updated to handle multiple Service Alternative Names. To provide multiple of them seperate them with `,`, example: `-servername localhost,additionalSAN`. For Windows variant the value needs to be enclosed in double quotes.
 
 ## Bash Shell Script Standalone (Linux, Mac OS X)
 
-When working on a *Linux* machine, shell script `ssl-tool/run.sh` can be used directly from command line. It's required to have `OpenSSL` and `keytool` programs available in the environment.
+When working on a *Linux* machine, shell scripts can be used directly from command line. It's required to have `OpenSSL` and `keytool` programs available in the environment.
```

Comment:
What version of OpenSSL and keytool?

Reply:
Would probably be the best to use latest version, I can add that mention here.